app/components/Reactions/index.tsx

```diff
@@ -101,16 +102,23 @@ class Reactions extends Component<Props, State> {
     } = this.props;
     const { reactionPickerOpen, addEmojiHovered } = this.state;
     return (
-      <div className={styles.reactionsContainer} ref={this.node}>
+      <div
+        className={styles.reactionsContainer}
+        ref={(node) => {
+          this.node = node;
+        }}
```

Comment:
This fixes the bug in which the reactions picker closes when trying to click on the categories.